doc/doxygen_project.txt.in

```diff
@@ -237,3 +237,9 @@ SEARCHENGINE           = $MRPT_USE_SEARCHENGINE
 # file to put on the web server. The advantage of the server based approach is that it scales better to large projects and allows full text search. The disadvances is that it is more difficult to setup
 # and does not have live searching capabilities.
 SERVER_BASED_SEARCH    = YES
+
+#---------------------------------------------------------------------------
+# Citation
+#---------------------------------------------------------------------------
+CITE_BIB_FILES = "../libs/vision/src/pnp/pnp_algos.bib"

```

Comment:
Shouldn't `bib` files belong much more naturally to `MRPT/docs` or so? I see very weird to place them under `*/src/`... please, change them with another commit in this PR
(cc: @jbriales )